pkg/cmd/roachtest/roachtestutil/mixedversion/planner_test.go

```diff
@@ -74,17 +75,16 @@ func TestTestPlanner(t *testing.T) {
 	// are added, and also allows us to test mutators explicitly and in
 	// isolation.
 	originalMutators := planMutators
-	resetMutators := func() { planMutators = nil }
 	defer func() {
 		planMutators = originalMutators
 	}()
 
 	datadriven.Walk(t, datapathutils.TestDataPath(t, "planner"), func(t *testing.T, path string) {
-		resetMutators()
+		eraseMutators()
```

Comment:
Nit: I actually liked `resetMutators` :) `eraseXXX` usually denotes something more involved than clearing a pointer. Maybe `clearMutators` instead?

Reply:
I'm okay with `resetMutators` :). I think the name is still "resetting" the mutators but `clearMutators` works with me too if others like that more.